.circleci/config.yml

```diff
@@ -52,18 +53,52 @@ commands:
     description: "Build"
     steps:
       - run:
-          name: "Grunt deploy"
-          command: npm run deploy-ci
+          name: "Compile project"
+          command: npm run build
+
+  compress:
+    description: "Compress"
+    steps:
+      - run:
+          name: "Compress"
+          command: |
+            pushd dist/compressed/
+            tar -cvf artifact.tar *
```

Comment:
Why don't we compress the tar file? it needs `z` for `zip`.

Reply:
We could also exclude `.git` directory here because the artifact won't use it anyway.